tests/test_bonfire_mcp/test_server.py

```
@@ -1,7 +1,7 @@
 """Tests for bonfire_mcp.server module — tool definitions and dispatch."""
 
 import pytest
-from unittest.mock import MagicMock, patch
+from unittest.mock import AsyncMock, MagicMock, patch
```

Flake8 (7.3.0)

[error] 'unittest.mock.AsyncMock' imported but unused

(F401)